api/jobs.yaml

```diff
@@ -201,85 +208,12 @@ definitions:
       labels:
         type: object
         description: Custom job labels with string values
```

Comment:
How'd all this other stuff move?

Reply:
Oops I copied some stuff from my PR with job models updates. Removed.